pkg/labels/labels_test.go

```diff
@@ -372,3 +373,19 @@ func TestLabels_GetFromSource(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkLabel_String(b *testing.B) {
+	l := NewLabel("io.kubernetes.pod.namespace", "kube-system", "k8s")
```

Comment:
Also, might be worth adding `b.ReportAllocs()` and `b.ResetTimer()`

Reply:
done!